Update smol-slime.md: Added information on what required to build smol slimes and list of community projects

Depact opened the pull request:
Added information on what required to build smol slimes and list of community projects

ShineBrightMeow reviewed (changes requested):
Some information seems redundant. As this is under the DIY section, some of this should be basic knowledge already or the visitor should be familiar from building ESP trackers. Just add to existing sections if needed.

The Case/PCB section can go lower down the page, giving the visitor a chance to get an introduction of the hardware, software, and firmware first.

This seems to be a copy and paste from Philip6294's message on Discord. As for Aed's case, the coin cell battery is not specified as the other batteries are. What about Lyall's case?

Depact commented:
@ShineBrightMeow thanks for the review! I pushed my review-based changes.
- The DIY section was deleted until I had a good way to return it.
- Changed community projects list to table and added information about some components where I found it specified. My change was so similar to Philip6294 because I thought it was a greatly structured message that sadly wasn't on wiki. 
- Added LyallUlric case.

ShineBrightMeow commented:
I think "Community Cases/PCBs" sounds better since it is the same project/tracker.

As for the USB column, I think it is a bit redundant since Lyall and Scawanf's build/case both have USB making every item Yes.

Please add "Edited by <Your Username>." at the bottom credits.

ShineBrightMeow commented on src/diy/smol-slime.md:
```diff
@@ -14,44 +14,62 @@ Interested, have questions, or issues with this project? Chat with us in ***#smo
+ - LIS3MDL (Not Tested)
+ - MMC5983MA
+
+### Push Button/Momentary Switch 
```
Maybe the Button, Switch, and Battery can be in a single table format too. It would reduce the vertical spacing between headers.

![image](https://github.com/user-attachments/assets/96c923da-7ecb-42fa-add9-73726021486c)

Depact replied:
I'm not sure that I'm getting a vision how it should look. So I'm not sure how to improve. From what I see, those parts are too distinct to be in one table.

ShineBrightMeow replied:
I am thinking maybe another header called "Other Components" and then you have the a table with Component/Description (or Purpose) and you can list the Button, Switch, and Battery. What do you think?

Depact replied:
It makes some sense since currently, those chapters are tiny, but I'd keep them separated to avoid any possible confusion.

ShineBrightMeow commented on src/diy/smol-slime.md:
```diff
@@ -387,15 +405,40 @@ Please open a Github Issue for firmware bugs/issues in the corresponding reposit
+### Firmware Links
+| Name                          | Links                                                               |
+| ----------------------------- | ------------------------------------------------------------------- |
+| SlimeVR nRF Receiver Firmware | - [Github](https://github.com/SlimeVR/SlimeVR-Tracker-nRF-Receiver) |
```
Removing all of the "-" from Links will look a bit cleaner. Same for the sections below.

ShineBrightMeow reviewed (changes requested):
Everything else looks accurate other than some formatting changes.

ShineBrightMeow reviewed (approved):
I agree with Scawanf. Some parts are out of the scope of the PR.

Depact commented:
@calliePepper are we good to merge? I got reviews.

Depact commented:
@ShineBrightMeow
@sctanf
This PR is merged. From work on this PR I think those things can be done on smol slimes docs. 
- Add testing components on DOA. 
- Mentions of ready to use parts for straps like gopro strap. 
- Overall more info on straps.
- Sourcing of components. 
- Rewrite receivers and trackers part for more consistency and detail. 
- Link to part of docs explaining what you get for different amount of trackers. 
- Better telegraph that you don't have to compile own firmware. 
- Overall restructure to steps to reduce entrance barrier. 
- Add images to community cases similar way big slimes do. 
- Put higher emphasis on community cases, since they contain solutions to huge part of headache. 
- Come up with standardised way of measuring magnetic interference and signal strength to make figuring out signal and accuracy issues more streamlined.
